lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -465,6 +465,13 @@ function dosomething_user_new_user($form, &$form_state) {
   global $user;
   $account = $user;
 
+  if (module_exists('dosomething_global') {
```

Comment:
oh, yeah so what I was saying on the walk, this won't be set if the user is under 13, maybe put this function above the return for that check?